packages/story-editor/src/components/panels/design/textStyle/test/textStyle.js

```diff
@@ -63,6 +64,7 @@ jest.mock('../../../../form/color/color', () => {
   const React = require('@googleforcreators/react');
 
   const _PropTypes = require('prop-types');
+  // eslint-disable-next-line prefer-arrow-callback -- False positive.
   const FakeControl = React.forwardRef(function FakeControl(props, ref) {
```

Comment:
<!-- potential concern, ask for clarification or investigation -->
**❓question:**
Curious, doesn't it work the same way here as elsewhere? (`forwardRef((props, ref) => {...})`)

Reply:
Then there will be a lint warning about a missing component display name.

Reply:
I see now what you meant and fixed the ESLint config & undid the other related changes.